io.github.deaen.awesomecalculator.yml

```diff
@@ -0,0 +1,33 @@
+id: io.github.deaen.awesomecalculator
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.godotengine.godot.BaseApp
+base-version: '4.4'
+command: godot-runner
+
+finish-args:
+  - --share=ipc
+  - --socket=x11 # <- Godot's wayland still doesn't support certain app functions
+  - --socket=pulseaudio
+  - --device=input
+  - --require-version=1.16.0
```

Comment:
```suggestion
```

Note that I am against it, but this will block a lot of Ubuntu and derivated users.